code/game/objects/items.dm

```diff
@@ -48,7 +48,7 @@
 
 	var/list/attack_verb // used in attack() to say how something was attacked "[x] has been [z.attack_verb] by [y] with [z]"
 
-
+	var/grindable_reagent //Null if you can't grind it up. Otherwise, what does this thing grind into?
```

Comment:
You wouldn't have been able to before. For instance, juicing only worked off an associative list and you could only associate one reagent with each thing you wanted juiced. Juicing is gone now, but it has been replaced by processing which works similarly.

If you want to get more than one reagent from an item, I suggest you fill the item's reagents datum with those chemicals and use the extract function.